SLA.R

```diff
@@ -28,4 +32,69 @@ split_by_date <- ggplot(data = sla, aes(n_Leaves, Leaf_Area_cm2, color = Date))
   facet_wrap(~Date)
 print(split_by_date)
 
+## Create an SLA column
+sla <- sla %>% 
+  mutate(specific_leaf_area = (Leaf_Area_cm2 / Leaf_Mass_g)) 
+head(sla)
+
+## Read in the inventory data
+inventory <- read.csv("ss-inventory.csv")
+summary(inventory)
+# Get rid of points where Tag is NA (look for row "by 1802", and don't include it)
+sla <- sla[!is.na(sla$Tag),]
+
+## Table of mean and sd DBH for alive maples 
+ACRU_alive <- inventory %>%
+  filter(Species == "ACRU", Alive == "Yes") %>% 
+  group_by(Plot, Species) %>% 
+  summarize(mn = mean(DBH), sd = sd(DBH))
+
+# Create new dataset with plot, species, and tag columns
+# JOIN!!! sla and inventory_small
+inventory_small <- inventory %>% 
+  select(Plot, Species, Tag) 
+sla_joined <- left_join(sla, inventory_small)
+
+## SLA by Plot
+sla_plot1 <- sla_joined %>% 
+  ggplot(aes(Species, specific_leaf_area, color = Position)) +
+  geom_point() +
+  geom_jitter() +
+  theme_bw() +
+  facet_wrap(~Plot) +
+  theme(axis.text.x = element_text(angle = 90)) +
+  labs(title = "Specific Leaf Area by Plot", y = "Specific Leaf Area", 
+       subtitle = "Lillie Haddock 6/21/19")
```

Comment:
Might want to consider removing the subtitle, if you use the script again for new data it'll be annoying to have to go in and change the date every time

Reply:
Agreed. Better though: `subtitle = paste("Lillie Haddock", date())`